examples/fluids/qfunctions/blasius.h

```diff
@@ -245,17 +215,18 @@ CEED_QFUNCTION(Blasius_Inflow)(void *ctx, CeedInt Q,
 
     // -- Momentum
     for (CeedInt j=0; j<3; j++)
-      v[j+1][i] -= wdetJb * (rho * u_normal * velocity[j] // interior rho
+      v[j+1][i] -= wdetJb * (rho * u_normal * s.Y.velocity[j] // interior rho
                              + norm[j] * P // mixed P
                              + viscous_flux[j]);
 
     // -- Total Energy Density
-    v[4][i] -= wdetJb * (u_normal * (E + P) + Dot3(viscous_flux, velocity));
+    v[4][i] -= wdetJb * (u_normal * (E + P) + Dot3(viscous_flux, s.Y.velocity));
```

Comment:
I added `FluxTotal_Boundary` to the `newtonian_state.h`, so that could be used here to simplify things a decent bit.